gc/base/TLHAllocationSupport.hpp

```diff
@@ -93,19 +93,25 @@ class MM_TLHAllocationSupport
 	MMINLINE void *getBase() { return (void *)_tlh->heapBase; };
 	MMINLINE void setBase(void *basePtr) { _tlh->heapBase = (uint8_t *)basePtr; };
 
-	/* CMVC 143597: setRealAlloc and getRealAlloc are fix to ensure we seal the
+	/* CMVC 143597: setRealTop and getRealTop are fix to ensure we seal the
 	 * TLH properly under frequent hook and unhook of ObjectAllocInstrumentable, which toggles
-	 * enable/disable of TLH. If TLH is disabled, realHeapAlloc holds the true TLH alloc ptr.
+	 * enable/disable of TLH. If TLH is disabled, realHeapTop holds the true TLH top ptr.
 	 */
-	MMINLINE void setRealAlloc(void *realAllocPtr) { _tlh->realHeapAlloc = (uint8_t *)realAllocPtr; };
-	MMINLINE void *getRealAlloc()
+	MMINLINE void setRealTop(void *realTopPtr) { _tlh->realHeapTop = (uint8_t *)realTopPtr; };
+	MMINLINE void *getRealTop()
 	{
-		if (NULL != _tlh->realHeapAlloc) {
-			return _tlh->realHeapAlloc;
+		if (NULL != _tlh->realHeapTop) {
+			return _tlh->realHeapTop;
 		} else {
-			return *_pointerToHeapAlloc;
+			return *_pointerToHeapTop;
 		}
 	};
+
+	/* @return the number of bytes, which are available for allocation in the TLH */
+	MMINLINE uintptr_t getRealSize() { return (uintptr_t)getRealTop() - (uintptr_t)getAlloc(); };
```

Comment:
would it make more sense to call it Unused or Remaining?

Reply:
@dmitripivkine again, please your opinion. 'Real' is still ok, though.

Reply:
I like Remaining, it explains meaning better. I am not strong on this

Reply:
please update this one as per Dmitri's suggestion